File: AquaSense/config/supabase.js
import { createClient } from '@supabase/supabase-js'

const supabaseUrl = 'yrbvvkfvmhtjgovziiai.supabase.co'
const supabaseKey = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InlyYnZ2a2Z2bWh0amdvdnppaWFpIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NDYyMDI5NjEsImV4cCI6MjA2MTc3ODk2MX0.MafN8wVQyXJ-ciwPi5LePbVegNYJr_vtL-v9vk7bTsE'

export const supabase = createClient(supabaseUrl, supabaseKey)

File: AquaSense/app/config/supabase.py
from supabase import create_client, Client
from .settings import SUPABASE_URL, SUPABASE_KEY
import logging

logger = logging.getLogger(__name__)

try:
    logger.debug(f"Initialisation de Supabase avec URL: {SUPABASE_URL}")
    supabase: Client = create_client(SUPABASE_URL, SUPABASE_KEY)
    logger.debug("Client Supabase initialisé avec succès")
except Exception as e:
    logger.error(f"Erreur lors de l'initialisation de Supabase: {str(e)}")
    raise 